tests/e2e/specs/add-paid-campaigns/add-paid-campaigns.test.js

```diff
@@ -409,107 +385,67 @@ test.describe( 'Set up Ads account', () => {
 
 				await expect(
 					page.getByText(
-						'In order to launch your paid campaign, your billing information is required. You will be billed directly by Google and only pay when someone clicks on your ad.'
+						'You do not have billing information set up in your Google Ads account. Once you have set up billing, you can start running ads.'
 					)
 				).toBeVisible();
-
-				await expect(
-					page.getByRole( 'link', {
-						name: 'click here instead',
-					} )
-				).toBeVisible();
```

Comment:
Is it necessary to remove this assertion?

Reply:
Not sure why it was removed in first place. I restored it 👍